doc/CMakeLists.txt

```diff
@@ -19,28 +19,36 @@
 
 if ( NOT CMAKE_CROSSCOMPILING )
   add_custom_target( generate_help ALL )
-  # Extract help from all source files in the source code, put them in
-  # doc/help and generate a local help index in the build directory containing 
-  # links to the help files.
-  add_custom_command( TARGET generate_help POST_BUILD
-    COMMAND python -B generate_help.py "${PROJECT_SOURCE_DIR}" "${PROJECT_BINARY_DIR}"
-    COMMAND python -B generate_helpindex.py "${PROJECT_BINARY_DIR}/doc"
-    WORKING_DIRECTORY "${PROJECT_SOURCE_DIR}/extras/help_generator"
-    COMMENT "Extracting help information; this may take a litte while."
-    )
-  # Copy the local doc/help directory to the global installation
-  # directory for documentation.
-  install( DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/help"
-    DESTINATION "${CMAKE_INSTALL_DOCDIR}"
-    )
-  # Update the global help index to contain all help files that are
-  # located in the global installation directory for documentation.
-  install( CODE
-    "execute_process(
-      COMMAND python -B generate_helpindex.py \"${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_DOCDIR}\"
-      WORKING_DIRECTORY \"${PROJECT_SOURCE_DIR}/extras/help_generator\"
-    )"
-    )
+
+    find_package( PythonInterp )
+    if ( PYTHONINTERP_FOUND )
+      if(${PYTHON_VERSION_STRING} VERSION_GREATER "2.7.7")
```

Comment:
I think it would be useful to add a comment explaining why we require a version > 2.7.8, including a reference to #678. Also, add spaces around the parenthesis `if ( ... )`.